Makefile

```diff
@@ -111,11 +99,23 @@ local-docker-run:
 			--namespacePollingInterval=10 \
 			--logLevel=debug
 
-# Test locally using kind
-local-test:
+# Run tests
+tests:
+	@echo "Running Unit tests..."
```

Comment:
When running tests, we are running all tests unit and e2e, so we should change this message.

Reply:
Actually, we are not running the old tests we have inside the `pkg/services`.

Reply:
I think this is a little bit more complex, but maybe we should be able to run unit tests only as well.

Reply:
Fixed on 18674a001f4262768c1676032b7e9064a50bc811 (PR #55). I checked that all unit tests are running (and only them).